### PR TITLE
Add hint to accept org invite for new members

### PR DESCRIPTION
**Pull request by dhiller**
Add a hint for new org members that they first need to accept the org invite before they become a full member.

/cc @aburdenthehand @alicefr @fabiand

### CODE REVIEW

File: membership_checklist.md [hunk 1]
```diff
@@ -28,3 +28,6 @@ e.g. (at)example_user
 -
 -
 -
+
+> [!IMPORTANT]
+> After this pull request has been merged, the automation will trigger sending an invitation to the KubeVirt GitHub organization to the new member. The new member needs to accept the invitation to receive member status.
```
Comment (aburdenthehand) [inline membership_checklist.md:33]:
```suggestion
> After this pull request has been merged, an invitation to the KubeVirt GitHub organization will be automatically sent to the new member. The new member needs to accept the invitation to receive member status.
```

File: membership_policy.md [hunk 2]
```diff
@@ -47,6 +47,9 @@ Defined by: Member of the KubeVirt GitHub organization
   * Have your sponsoring org members reply confirmation of sponsorship: +1
   * Once your sponsors have responded, your request will be reviewed by KubeVirt project approvers. Any missing information will be requested.
 
+> [!IMPORTANT]
+> After the pull request against the org members section has been merged (see above), the automation will trigger sending an invitation to the KubeVirt GitHub organization to the new member. The new member needs to accept the invitation to receive member status.
```
Comment (aburdenthehand) [inline membership_policy.md:51]:
```suggestion
> After the pull request against the org members section has been merged (see above), an invitation to the KubeVirt GitHub organization will be automatically sent to the new member. The new member needs to accept the invitation to receive member status.
```

**Review by aburdenthehand — COMMENTED** [review]
Looks great but I have suggested a couple of changes to wording (same change in both instances).

**Comment by dhiller** [comment]
@aburdenthehand thank you very much for the feedback! PTAL!

**Review by brianmcarey — COMMENTED** [review]
/lgtm

**Pull request merged**